cadl-tests/package.json

```diff
@@ -14,7 +14,8 @@
     "@azure-tools/cadl-ranch-expect": "latest",
     "@azure-tools/cadl-ranch-specs": "latest",
     "@azure-tools/cadl-ranch-api": "latest",
-    "@azure-tools/cadl-java": "file:/../cadl-extension/azure-tools-cadl-java-0.1.0-dev.7.tgz"
+    "@azure-tools/cadl-java": "file:/../cadl-extension/azure-tools-cadl-java-0.1.0-dev.7.tgz",
+    "@cadl-lang/openapi": "latest"
```

Comment:
Please group it to @cadl-lang, and use the ~version, not the latest. We only plan to upgrade it when we upgrade compiler.

Reply:
updated